scripts/generateServerConfig.js

```diff
@@ -1,6 +1,6 @@
 
 import * as fs from 'fs';
-import("./generateClientConfig.js");
+// import("./generateClientConfig.js");
```

Comment:
Can this comment be removed, and can `generateClientConfig` be deleted if it still exists